bccsp/pkcs11/impl_test.go

```diff
@@ -743,7 +743,10 @@ func TestECDSAKeyImportFromECDSAPublicKey(t *testing.T) {
 		t.Fatalf("Failed getting ECDSA raw public key [%s]", err)
 	}
 
-	pub, err := utils.DERToPublicKey(pkRaw)
+	if len(pkRaw) == 0 {
+		t.Fatalf("Invalid DER. It must be different from nil")
+	}
```

Comment:
I don't understand why this validation is needed. ParsePKIXPublicKey should return an error. A test will let us know if it's needed or not.